app/web/package.json

```diff
@@ -14,6 +14,7 @@
     "format:check": "prettier --check '**/(*.ts|.tsx)'",
     "jest": "jest --modulePaths=.",
     "lint": "next lint",
+    "lint:fix": "next lint --fix .",
```

Comment:
It was annoying me that there was only one to see the incorrect linting but not autofix it. There's an autofix for the backend so added this. Let me know if that's bad.